src/main/java/au/gov/nla/dlir/models/event/Correction.java

```diff
@@ -0,0 +1,15 @@
+package au.gov.nla.dlir.models.event;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class Correction {
+  private String type;
+  private Integer lineNumber;
+  private String correctedText;
+  private Integer beforeLineNumber;
+}
```

Comment:
Is this Correction just for the event processing? Does it need an id or some sort of timestamp?